### ISSUE TITLE
🚀 Feature Request: Container - Add CSS Grid options

### ISSUE THREAD

**Issue by Arian-c27** [elementor/elementor#19914]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It would be nice if you would add a CSS grid mode to container options to make it more flexible. It's especially tricky when creating layouts with multiple rows and multiple columns. 
I've made the video explaining the issue and proposing a solution

### Describe the solution you'd like

https://www.youtube.com/watch?v=5IhsmN-8GU0

**Comment by black-eye** [comment]
+1

**Comment by nicholaszein** [comment]
Hello @Arian-c27,

thanks for submitting your feature request! 🙏 

We will consider your request. But for now, we are focusing on releasing a stable Flex Container first.

Please understand that while we do welcome new feature requests, we can't promise they will be considered or added to our roadmap anytime soon. 

Best regards

**Comment by mkcam1** [comment]
@Arian-c27 I can almost sign that Elementor will never have this feature, the hopes now are Bricks Builder.

**Comment by 79ho3ein** [comment]
> @Arian-c27 I can almost sign that Elementor will never have this feature, the hopes now are Bricks Builder.
> 
> https://bricksbuilder.io/roadmap/

Exactly. They added conditional logic yesterday. I have been migrating to BRICKS BUILDER for almost a week now.

**Comment by Arian-c27** [comment]
> @Arian-c27 I can almost sign that Elementor will never have this feature, the hopes now are Bricks Builder.
> 
> https://bricksbuilder.io/roadmap/

Unfortunately as a theme developer i can’t ask my users to buy a paid plugin. But Elementor could catch up with Bricks very fast if they focus on the right things. 

- Fix the rendering which is the #1 issue by far why this editor struggles with speed. Right now we are overwriting it to solve some of the main issues but this causes us a lot of problems with updates
- Add essential features like CSS grid. It could be added and tested in a few hours if they focus in it. 

The weird thing is they are using things like row-gap with flex display which causes more incompatibility issues with older browsers than using grid display and gris-gap

**Comment by 79ho3ein** [comment]
I quite agree @Arian-c27, Elementor is still the best page builder. But if it doesn't take advantage of its current position, another page builder will take its place in less than a year. I have been working with Elementor for 4 years and I really love it. But to be honest, Elementor's progress is disappointing. At the moment, I am completely proficient in bricks builder. But it's hard for me to get away from Elementor. Because Elementor is my favorite page builder.
I don't know, maybe the Elementor team has a plan to increase the speed of their plugin development. If so, I hope they hurry. before it's too late...

**Comment by nicholaszein** [comment]
Thanks for your feedback @mkcam1 and @79ho3ein!

As I mentioned, we are focusing on releasing a stable Loop Builder using flexbox first. Grid is something we want to add, but not until we have addressed all the rough edges of the Flex Container, loop builder and other related features. This request is being considered, but only for future releases.

**Comment by luisceladita** [comment]
Come on, Flexbox and CSS Grid containers are so easy, hire me to implement Flexbox and Grid layout to Elementor lol.

**Comment by arielk** [comment]
I merged this issue with another issue https://github.com/elementor/elementor/issues/18839 in the same subject so we could deal with it more efficiently. So please let's continue the conversation there.

Thanks for your feedback!

**Comment by nicholaszein** [comment]
Hello, there everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the first phase of this feature was released with Elementor **v3.13.0**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
